src/main/java/org/reaktivity/nukleus/tls/internal/stream/TlsClientFactory.java

```diff
@@ -170,6 +174,7 @@ public TlsClientFactory(
         this.correlations = new Long2ObjectHashMap<>();
         this.decodeBudgetMax = decodePool.slotCapacity();
         this.handshakeBudgetMax = Math.min(config.handshakeWindowBytes(), decodeBudgetMax);
+        this.handshakeTimeout = config.handshakeTimeout();
```

Comment:
Rename to `handshakeTimeoutMillis` so we can convert `SECONDS.toMillis(config.handshakeTimeout())` just once.